docs/schedule.md

```diff
@@ -1,35 +1,35 @@
+|              | 11:00 PM       | Venue Closes                  | Niteo House                                         |
+| **Tuesday**  | 8:00 AM        | Venue Opens                   | Niteo House                                         |
+|              | 9:00 AM        | Breakfast                     | Niteo House                                         |
+|              | 12:00 PM       | Lunch Catering                | Niteo House                                         |
+|              | 2:00 PM        | Afternoon Activities          | Offsite                                             |
+|              | 7:00 PM        | Dinner Catering               | Niteo House                                         |
+|              | 11:00 PM       | Venue Closes                  | Niteo House                                         |
+|**Wednesday** | 8:00 AM        | Venue Opens                   | Niteo House                                         |
+|              | 9:00 AM        | Breakfast                     | Niteo House                                         |
+|              | 12:00 PM       | Lunch Catering                | Niteo House                                         |
+|              | 2:00 PM        | Group Activity / Trip         | Offsite                                             |
+|              | 7:00 PM        | Dinner Party                  | La Ola https://maps.app.goo.gl/yxTBkXBpZdaKxZ9s5    |
+| **Thursday** | 8:00 AM        | Venue Opens                   | Niteo House                                         |
+|              | 9:00 AM        | Breakfast                     | Niteo House                                         |
+|              | 12:00 PM       | Lunch                         | Niteo House                                         |
+|              | 2:00 PM        | Afternoon Activities          | Offsite                                             |
+|              | 7:00 PM        | Dinner Catering               | Niteo House                                         |
+|              | 11:00 PM       | Venue Closes                  | Niteo House                                         |
+| **Friday**   | 8:00 AM        | Venue Opens                   | Niteo House                                         |
+|              | 9:00 AM        | Breakfast                     | Niteo House                                         |
+|              | 12:00 PM       | Lunch Catering                | Niteo House                                         |
+|              | 3:00 PM        | Venue Cleaning                | Niteo House                                         |
+|              | 4:00 PM        | Venue Closes                  | Niteo House                                         |
+|              | 7:00 PM        | Farewell Party                | Pizza Time https://maps.app.goo.gl/eCo6HebYCdD6aUs49|
```

Comment:
In https://github.com/OceanSprint/organizing/issues/41#issuecomment-2646553699 I wrote that I agreed pizza at 6PM.